src/amo/containers/Home.js

```diff
@@ -34,6 +34,21 @@ export class HomePageBase extends React.Component {
     const { i18n } = this.props;
     return (
       <div className="HomePage">
+        <div className="HomePage-welcome">
+          <p className="HomePage-welcome-text">
+            {i18n.gettext(dedent`Extensions are special features you can add to Firefox.
+            Themes let you change your browser's appearance.`)}
+          </p>
+          <div className="HomePage-welcome-links">
+            <Link className="HomePage-extensions-link" to="/extensions/">
+              {i18n.gettext('extensions')}
```

Comment:
Should this be lowercase? I guess it doesn't matter as the CSS styles it to drop-caps but it seems like it should be capitalised.